avm/res/document-db/database-account/tests/e2e/analytical/main.test.bicep

```diff
@@ -17,16 +17,15 @@ param serviceShort string = 'dddaanl'
 @description('Optional. A token to inject into the name of each resource.')
 param namePrefix string = '#_namePrefix_#'
 
-// Pipeline is selecting random regions which dont support all cosmos features and have constraints when creating new cosmos
-#disable-next-line no-hardcoded-location
-var enforcedLocation = 'eastasia'
+@description('Optional. The location to deploy resources to.')
```

Comment:
Please consider that without an enforced location, the location will be randomly chosen among the intersection of allowed locations and this list of default ('eastus', 'uksouth', 'northeurope', 'eastasia').
You can check what this intersection would be from the logs. Before changing this we should make sure that any of the locations from that list would work

Reply:
@eriqua, right now `eastasia` and `northeurope` are throwing capacity errors for Azure Cosmos DB in my pipeline.

Would it be preferable to change the enforced location to something else? Can I choose a location outside of the allowed locations list?

CC: @bryansan-msft

Reply:
For now, I changed it to `eastus2` and will change it to something else if your team advises. This is the region I use in my own testing.

Reply:
You can @seesharprun. Think of the allowed list more like a list common to the majority of services. You surely can enforce a different one on your tests, even a different one for each test if that is useful .
You can resolve this comment once read 🙂